packages/integration-karma/test/light-dom/basic/index.spec.js

```diff
@@ -0,0 +1,40 @@
+import { createElement, setFeatureFlagForTest, LightningElement } from 'lwc';
```

Comment:
Until we add support for slotted content we should throw at runtime when some content is slotted into a light DOM component. It doesn't need to be part of this PR, but it's something we should prevent before the release next week.